api/proto/banyandb/database/v1/schema.proto

```diff
@@ -116,14 +135,39 @@ message Measure {
     repeated FieldSpec fields = 3;
     // entity indicates which tags will be to generate a series and shard a measure
     Entity entity = 4;
-    // shard_num is the number of shards
-    uint32 shard_num = 5;
-    // ttl indicates time to live, how long the data will be cached
-    Duration ttl = 6;
+    // interval_rules define data points writing interval
+    repeated IntervalRule interval_rules = 5;
```

Comment:
@wu-sheng As we discussed, interval_rules instrument how frequently data points in a series are written? @lujiajing1126 we should add a new tag in the java client to match these rules.